### PR TITLE
Metastation: Maintenance Rework [Part 2]

### PR DESCRIPTION
**Pull request by Venuska1117**
**I Request TM on this, so i can find issues quicker.**

## What Does This PR Do
Expand maintenances and add little more loot, change some room shapes and orientation to match each others without breaking consistency.

## Why It's Good For The Game
More areas to walk without suffocating, and more optimal room placements (not all but most of them)

## Images of changes
I will keep updating this image if i made changes. [14.12.2023]
StrongDMM-2023-12-14 01 13 28

## List of stuff i did, because it hit [MQL] without any feedback on github.

- Changed shape of connected kitchen and bar, now they are re-shaped, it was done this way so botany can be connected to kitchen.

- Xenobiology cage (one at back of xenobiology) with probably was intended to be used by sciencist to test chemicals, now was changed to be dedicated containment cell.

- Chemist room now have its own testing room with monkey, its mostly copy from cyberiad.

- Some of science rooms were moved around to increase maintenance space.

- The xenobiology diff is because first, i edited it so it was connected to main loop so antags spawning in vents (xeno, terrors, etc.) will not get stuck there, after that i did other edits, and day later i got informed to not connect it to main loop, so now it have some edits and diff.

## Testing
Run VSC and it work for now.

## Changelog
:cl: Venuska1117
tweak: Rework and Expand Metastation: Starboard Maintenance, Starboard Secondary Maintenance, Alf-Starboard Maintenance and  Alf Secondary Maintenance.
/:cl:

### CODE REVIEW

**Comment by Venuska1117** [comment]
I will say it before someone check it, i did some changes with can touch other parts of station in map diff, its because i edited path for cables/pipes, because there was blue pipe with layer 2 (i didnt make it, and missed that in previous PR) and horizontal cable with dir 4 and 8 (they both Look same) so i swapped all 8 to 4dir.... Ah and Universal enzymes in maintenance for some reason have layer 5 (?) so i swapped it to normal one.

**Review by Henri215 — COMMENTED** [review]
image

1. You ended up removing that research monitor from xenobio?

image


2. I would suggest adding a pen and a old drip of blood on a tile next to the paper, adds more to the immersion.

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -576,6 +576,10 @@
...
+/obj/item/paper/crumpled/bloody/hacker
```
Comment (Sirryan2002) [inline code/modules/paperwork/paper.dm:579]:
```suggestion
	info = "As the Diona species, we awoke aboard our terraformation vessel with the primary goal of reshaping the alien world. Our endeavors were highly successful, as we cultivated various plant species and made astonishing discoveries throughout our journey. We seeded a remarkable 'special' grass around our ship, which thrived splendidly. However, as time passed, we faced a growing challenge - a shortage of oxygen in our containment tanks hindered our ability to spread the grass further. In response, we embarked on a series of trials and experiments to engineer plants with the capacity to survive in low-oxygen environments, thus extending our breath of life. <br>Through a series of trials, combining failures and successes, we unveiled several plant species with unique attributes. Some proved to be valuable for healing purposes, while others offered addictive properties. Glowing mushrooms emerged as a source of vital light, preventing us from succumbing to the darkness. Among these discoveries, one plant commanded our utmost attention – the 'space tobacco.' While this species did not generate oxygen, it contained a chemical known as Salbutamol, enabling us to respire in low-oxygen conditions when consumed. The only drawback was the need to meticulously extract harmful compounds for its safe utilization. <br>Amid our efforts to expand the greenery, an unexpected encounter transpired. I found myself under assault by an enigmatic creature, and I was forced to flee in haste, straying too far from our vessel. As I stand now, my supplies of life-sustaining plants are dwindling, as is my ability to endure in this low-oxygen environment. Suffocation looms, and I must hasten my return to the safety of our ship to avert this dire fate."
	
/obj/item/paper/crumpled/bloody/hacker
```
spacing

**Review by S34NW — CHANGES_REQUESTED** [review]
image
This concerns me greatly, do both servers work as intended or do we run into issues when one is shut off, destroyed, etc.

image
Nice electrified grilles bro

image
Random wall pipe south of this cooler

image
Unconnected air vent here

image
no access on this door meaning anyone can nick this stuff, intended?

image
No hydroponics access on this door, but there is on the maint door.

image
Missing manifold here

**Comment by DGamerL** [comment]
> This concerns me greatly, do both servers work as intended or do we run into issues when one is shut off, destroyed, etc.

Why is there even a second, hidden RnS server there? Wouldn't this make destroying the server in science obsolete?

**Comment by Henri215** [comment]
> > This concerns me greatly, do both servers work as intended or do we run into issues when one is shut off, destroyed, etc.
> 
> Why is there even a second, hidden RnS server there? Wouldn't this make destroying the server in science useless?

Reason is simple, decoration.

**Comment by Venuska1117** [comment]
> image This concerns me greatly, do both servers work as intended or do we run into issues when one is shut off, destroyed, etc.
> 
> image Nice electrified grilles bro
> 
> image Random wall pipe south of this cooler
> 
> image Unconnected air vent here
> 
> image no access on this door meaning anyone can nick this stuff, intended?
> 
> image No hydroponics access on this door, but there is on the maint door.
> 
> image Missing manifold here

I dont know what to do about botany first door, before rework first door didnt have access, so everyone can access "public" vendors (botany have its own vendors)

**Review by Aylong220 — COMMENTED** [review]
DiffBot shows that almost the entire map is affected, I don't think that's how it was designed, if I were you I would copy all my work into a separate file, take the map from the master branch, and overlay the previously saved work into a separate file, this will minimize the number of conflicts for future PRs, and make DiffBot cleaner
image

I think changing this place is beyond the purpose of the PR, but come on, all I can say is that it's a strange decision to take away the tables and components from the room on the right side of the room
image

There was a spacewalk here, is there some reason it was removed?
image

I'm shocked at the amount of pipes in the walls, it's a violation of standards and just a headache if you have to change them out
image

I think it would be better to just put tinted windows, although that's not the best solution either, but I don't see any options to make it better without rebuilding everything
image

APC can be moved to a more accessible location and without wiring under reinforced tables
image

image

Status display under window and light in air
image

You forgot to turn shutters?
image

Exo fab in maint? Strange for me
image

image

Why are there so many vents on pipes?
image

I'm certainly not from balance team, but I think hydro with biogen in maints is a bit much
image

Why not put a normal door here, and where Pun-Pun? 
Plus about the kitchen, I think her floor used to be better, more unique
image

1 scrubber and 1 vent is very little for room like this
And I don't understand the need for a circle of wires
image

Metal rods in wall
image

Other than that, everything else looks fine to me

**Comment by S34NW** [comment]
> 
> I dont know what to do about botany first door, before rework first door didnt have access, so everyone can access "public" vendors (botany have its own vendors)

Fine to leave as-is then, thanks for checking

**Review by S34NW — APPROVED** [review]

**Pull request merged**